### PR TITLE
updated structure and wording 

### PR DESCRIPTION
**Pull request by kiwibayer**
## Overview/Summary

Updated the structure and wording 

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

## As part of this Pull Request I have

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md [hunk 1]
```diff
@@ -228,35 +238,38 @@ Click **Next: Landing Zone configuration>** to continue with your deployment.
 
 ![identityTab-next](./media/clip_image036c-1-singlesubscription.png)
 
-## Landing zones configuration
+## 9. Landing zones configuration
 
-You can optionally bring in N number of subscriptions that will be bootstrapped as landing zones, governed by Azure Policy:
+It is posibly to bring in N number of excisting subscriptions that will be bootstrapped as landing zones, governed by Azure Policy:
```
Comment (juazasan) [inline docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:243]: Typo: excisting --> existing

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md [hunk 2]
```diff
@@ -42,32 +43,41 @@ Click **Next: Azure Core Setup>** once you had chosen your deployment Region.
...
 On the *Azure Core setup* blade you will:
 
-- **Provide a prefix** that will be used to name your management group hierarchy and platform resources.
-- Choose between using dedicated subscriptions or a single subscription to host platform resources. Please note that dedicated subscriptions are recommended.
+- **Provide a prefix** that will be used to name your management group hierarchy **and** platform resources.
+- Choose between using dedicated subscriptions or a single subscription to host platform resources. 
 
-  For this scenario select **Single** and **provide a dedicated (empty) subscription** that will be used to host your Platform resources.
+**Please Note:** A dedicated platform subscriptions is in general recommended. However, some Customers have the requirement to host their plattform and applications wihtin a single subscription. The Trey Research Reference is aimed at Customers with this requirement. 
+
```
Comment (juazasan) [inline docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:54]: We are willing to reduce coupling between the deployment guide (s) and the RI (s) as much as possible as customers may end up in the guide without having gone through the RI. We have a brief reference in the beginning of each guide but then we want customers to be able to complete the deployment without "knowing" what TR or Contoso or Adventure Works are.
Hence, I would prefer to replace Trey Research with "this scenario" or "this tutorial".

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md [hunk 3]
```diff
@@ -42,32 +43,41 @@ Click **Next: Azure Core Setup>** once you had chosen your deployment Region.
...
+  For the Trey Research scenario select **Single** and **provide a dedicated (empty) subscription** that will be used to host your Platform resources.
```
Comment (juazasan) [inline docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:55]: Same as previous comment.

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md (hunk 2 above)
Comment (kiwibayer) [reply docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:54]: thanks for explaining, that's fine with me

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md (hunk 3 above)
Comment (kiwibayer) [reply docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:55]: thanks for explaining, that's fine with me

**Pull request merged**